onyo/lib/onyo.py

```diff
@@ -576,6 +604,29 @@ def _init(self, directory: Union[Path, str]) -> None:
         from .command_utils import init_onyo
         return init_onyo(self, directory)
 
+    @staticmethod
+    def _is_onyo_root(directory: Path) -> bool:
+        """
+        Assert whether a directory is the root of a repository and has a fully
+        populated `.onyo/` directory.
+        """
+        dot_onyo = Path(directory, '.onyo')
+
+        if not dot_onyo.is_dir() or \
+                not Path(dot_onyo, "templates").is_dir() or \
+                not Path(dot_onyo, "validation").is_dir() or \
+                not Path(dot_onyo, "config").is_file() or \
+                not Path(dot_onyo, ".anchor").is_file() or \
+                not Path(dot_onyo, "templates/.anchor").is_file() or \
+                not Path(dot_onyo, "validation/.anchor").is_file():
+            return False  # noqa: E111, E117
```

Comment:
Why `# noqa: E111, E117`?

E111 is when indentation is not a multiple of four, and E117 is when something is over-indented. This can easily be addressed, and shouldn't even trigger on this line. If the shortened solution above is used, then this will become more obvious.

Reply:
+1

I didn't comment on it, because I haven't dived much into the linter setup yet, but I did wonder, too. If those errors are what @nhjjr is saying, then I agree.